src/use_sinon_sandbox.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-env goodeggs/server-side-test */
+
+import sinon from 'sinon';
+import assert from 'assert';
+
+export default function useSinonSandbox () {
+  beforeEach('use sinon sandbox', function () {
+    this.sinon = sinon.sandbox.create({
+      injectInto: this,
+      properties: ['spy', 'stub', 'mock'],
+      useFakeTimers: false,
+      useFakeServer: false,
+    });
+
+    const clock = this.clock = this.sinon.useFakeTimers({
+      now: 0,
+      toFake: ['Date'],
+    });
+
+    this.stubTime = function stubTime (date) {
+      assert(date != null, 'date required');
+      clock.setSystemTime(date);
+      return date;
+    };
+
+    this.stubLogger = function ({logger} = {}) {
```

Comment:
You've got an empty `src/stub_logger.js` file FYI... I imagine this had to be here because it uses the sinon sandbox to stub. That seems ok.

Reply:
Good catch! ⚾️

Reply:
Did you intentionally change the signature from `stubLogger(logger)` to stubLogger({logger}`? I think this is a case where an unnamed argument is what most people would expect. Also doesn't seem seem worth all the refactoring here?

Reply:
Funny, I was also just bitten by this.

Reply:
It was intentional, but in retrospect, probably not good change. I will change it back and publish a new major version.

Reply:
@demands @serhalp fixed in `goodeggs-test-helpers@5.0.0`